packages/material-ui/src/TextareaAutosize/TextareaAutosize.test.js

```diff
@@ -79,8 +79,8 @@ describe('<TextareaAutosize />', () => {
       it('should handle the resize event', () => {
         const wrapper = mount(<TextareaAutosize />);
         assert.deepEqual(getStyle(wrapper), {
-          height: undefined,
-          overflow: null,
+          height: 0,
+          overflow: 'hidden',
```

Comment:
I don't think there should be any changes here, So I think maybe We need to come up another logic.
I guess `overflow` should be `null` here.

What are your thoughts?